websites/G/Goodreads/presence.ts

```diff
@@ -1,7 +1,6 @@
 const presence = new Presence({ clientId: "867525909204566056" }),
-	timeElapsed: number = Math.floor(Date.now() / 1000);
-
-let book: string, author: string;
```

Comment:
We actually don't need these vars. This was only putted here in the past only for readability.